libraries/SITL/examples/Airsim/quadX.parm

```diff
@@ -8,14 +8,14 @@ AUTOTUNE_AXES    1.000000
 DISARM_DELAY     0.000000
 FRAME_TYPE       1.000000
 FS_CRASH_CHECK   0.000000
-GND_ABS_PRESS    94493.945312
-INS_GYR_CAL      0.000000
+GND_ABS_PRESS    94499.976562
```

Comment:
why set baro pressure and gyro offsets? These are set automatically on each boot

Reply:
Wasn't sure if I should keep them or not
Will remove them

Reply:
Done!